bans-core/src/main/resources/lang/messages_zh_tw.yml

```diff
@@ -0,0 +1,366 @@
+#Messages configuration
+#In most cases, the variables inside the default messages are those available
+#in that specific message. The exception to this is messages which are related
+#to a certain punishment.
+#When message has an associated punishment, multiple variables are available:
+#%ID% - punishment ID number
```

Comment:
Why were the spaces before and after the comment character removed? It used to look like this:
```
 # When message has an associated punishment, multiple variables are available:
 # %ID% - ...
```

Now it looks like this:
```
#When message has an associated punishment, multiple variables are available:
#%ID%
```
Note that the space before the `#` and the space after the `#` are missing.

Reply:
hmm, maybe because i using crowdin to translate, so it auto remove, i will try to add back

Reply:
OK, I add all back